kobo/settings/testing_no_stripe.py

```diff
@@ -0,0 +1,4 @@
+from .testing import *  # noqa
+STRIPE_ENABLED = False
+INSTALLED_APPS = tuple(filter(lambda x: x not in ['djstripe', 'kobo.apps.stripe'],
+                              INSTALLED_APPS))  # noqa
```

Comment:
Does not seem to be formatted with Black? 
It gives me 

```python 
INSTALLED_APPS = tuple(
    filter(lambda x: x not in ['djstripe', 'kobo.apps.stripe'], INSTALLED_APPS)
)
```

Why is the 2nd `noqa` needed?